src/pages/admin/pullrequests/review.astro

```diff
@@ -39,7 +41,9 @@ const prs: PullRequest[] = (await prisma.pullRequest.findMany({
     where: {
         PullRequestStatus: {
             reviewed: false
-        }
+        },
+        drafts : false
```

Comment:
`drafts` is a typo and also not a column in our database.

Reply:
will try coming up with some other approach!